src/aap_eda/tasks/ruleset.py

```diff
@@ -32,17 +32,29 @@
 logger = logging.getLogger(__name__)
 
 
-def activate(activation_id: int, requester: str = "User") -> Job:
-    job_id = f"activation-{activation_id}"
-    return unique_enqueue(
-        "activation", job_id, _activate, activation_id, requester
+def activate(
+    activation_id: int, requester: str = "User", reconnect: bool = False
+) -> None:
+    job_id = _activation_job_id(activation_id)
+    unique_enqueue(
+        "activation", job_id, _activate, activation_id, requester, reconnect
     )
 
 
-def _activate(activation_id: int, requester: str = "User") -> None:
+def _activation_job_id(activation_id: int) -> str:
```

Comment:
The Activation model represents data persisted in database. Here `activation_job_id` is the rq job id. I feel it is outside the model. The method is private because the job id is used only by methods in this module. We can move it to a common place if other module also needs to access it.

Reply:
I see your point. This is a common topic of discussion in software design. 
In my understanding a model is a representation of a domain entity that has both stored data and behaviors associated with that entity, serving as a bridge between the database and application logic. It can contain persistent attributes (stored in the database), computed attributes (would be this case), and methods that aren't stored but are derived or executed at runtime. In MVC architectures is quite usual, other architectures like domain-driven put a stronger emphasis on separate the business logic from the data model but we don't have such a pattern or entity for activations that operates on our business logic. Maybe we should have it. 


As it is now I consider 'job_id' intrinsic to the activation, given its pattern and the dependency of the application logic on it. I’m open to involving more team members for their insights to reach a consensus.  @mkanoor @hsong-rh @Dostonbek1 @benthomasson 

(I won’t hold back the PR over this; it’s a dialogue worth having, not a blocker.)